Add Automatic Status Update docs (#1260)
kyma-incubator/compass: docs/director/automatic-status-update.md

# Automatic status update

> Applications and Runtimes in Director use the `Status` field to inform the user about their condition. The field can be 

Reviewer: ```suggestion
Applications and Runtimes in the Director use the `Status` field to inform the user about their condition. When a paired Application or Runtime communicates with the Director API for the first time, the Director automatically sets its status to **CONNECTED**. You can also manage statuses manually by updating the objects.
```

> set externally by updating the object, but it is also automatically set by the Director.

Reviewer: ```suggestion

```

> Every time an Application or Runtime calls the API, it's status is automatically set to `CONNECTED` regardless of the previous one.

Reviewer: ```suggestion

```
